src/pages/home/report/ReportActionsView.tsx

```diff
@@ -126,13 +130,21 @@ function ReportActionsView({
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
     }, [route, reportActionID]);
 
+    const shouldBuildOptimisticCreatedReportAction = useMemo(() => !allReportActions?.find((action) => action.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED), [allReportActions]);
```

Comment:
```suggestion
    const shouldBuildOptimisticCreatedReportAction = useMemo(() => !allReportActions?.findLast(isCreatedAction), [allReportActions]);
```

Created action is usually the last action and this can be more performant for reports where report actions can change frequently.